src/lib/annotations/Annotator.js

```diff
@@ -263,6 +263,11 @@ class Annotator extends EventEmitter {
 
         this.destroyPendingThreads();
 
+        if (this.createHighlightDialog.isVisible) {
```

Comment:
there's actually a few places where we just check `if(this. createHighlightDialog) {}` would you be able to swap those out for a check for whether or not the dialog is visible?

Reply:
Just noticed that you caught one of them below! So perfect :)